tests/test_file_handler.py

```diff
@@ -28,7 +28,7 @@ def test_add_file(mocker):
     # Act
     filepath = add_file("/tmp", "test.txt", "test")
     # Assert
-    assert filepath == "/tmp/test.txt"
+    assert filepath == "/tmp\\test.txt"
```

Comment:
This is fine and works on Mac/Linux. Majority audience are Unix based so it's OK.